packages/shared/src/internalApi/api.ts

```diff
@@ -79,6 +79,10 @@ export async function fetchEndpoint<T>(
     }
   }
 
+  if (process.env.NODE_ENV === 'development') {
+    console.log('Request to internal API: ', requestUrl.toString(), requestInit);
```

Comment:
I'd much rather use a logger with loglevels or leave this out entirely, attaching a debugger should solve most of the cases where this'll be useful imo.

Reply:
I tend to agree with this. Ok, let's remove this line